### DISCUSSION TITLE
Displaying a sponsor button in your repository

### DISCUSSION THREAD

**Discussion by anehzat** [community/community#80846] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

What's the process for getting https://thanks.dev/ in github's FUNDING.yml 
We have a few people in the community asking for this feature.

**Comment by varunpats** [comment]
Hey, please refer to [this doc](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository). It has step by step process of displaying a sponsor button in your repository. Hope this is the one you are looking for.

**Reply by balupton** [reply]
@varunpats the request here is for ThanksDev to be one of the officially supported platforms for the FUNDING.yml format

**Comment by pboling** [comment]
I just asked in the thanks.dev discord what the format of `USERNAME` is supposed to be, as I couldn't figure it out, and it is not documented anywhere.
https://discord.com/channels/1026737794187526145/1308751517599141938

The answer was, given that my github username is `pboling`:
```
thanks_dev: u/gh/pboling
```

Which results in a sidebar link that takes a user to:
https://thanks.dev/u/gh/pboling

**Reply by Sistemcat** [reply]
> I just asked in the thanks.dev discord what the format of `USERNAME` is supposed to be, as I couldn't figure it out, and it is not documented anywhere. https://discord.com/channels/1026737794187526145/1308751517599141938
> 
> The answer was, given that my github username is `pboling`:
> 
> ```
> thanks_dev: u/gh/pboling
> ```
> 
> Which results in a sidebar link that takes a user to: https://thanks.dev/u/gh/pboling

$15

**Reply by Sistemcat** [reply]
> I just asked in the thanks.dev discord what the format of `USERNAME` is supposed to be, as I couldn't figure it out, and it is not documented anywhere. https://discord.com/channels/1026737794187526145/1308751517599141938
> 
> The answer was, given that my github username is `pboling`:
> 
> ```
> thanks_dev: u/gh/pboling
> ```
> 
> Which results in a sidebar link that takes a user to: https://thanks.dev/u/gh/pboling

$15

**Reply by pboling** [reply]
What does `$15` mean?  And why have you posted it 3 times?

**Comment by Sistemcat** [comment]
> ### Select Topic Area
> General
> 
> ### Body
> What's the process for getting https://thanks.dev/ in github's FUNDING.yml We have a few people in the community asking for this feature.

15

> ### Select Topic Area
> General
> 
> ### Body
> What's the process for getting https://thanks.dev/ in github's FUNDING.yml We have a few people in the community asking for this feature.

$15

**Reply by pboling** [reply]
Why do you keep posting the same "15"?

**Comment by Sistemcat** [comment]
> ### Select Topic Area
> General
> 
> ### Body
> What's the process for getting https://thanks.dev/ in github's FUNDING.yml We have a few people in the community asking for this feature.

$15

**Reply by pboling** [reply]
Spam?  Why?
